python/pipeline/experiment.py

```diff
@@ -6,7 +6,7 @@
 from commons import lab
 
 
-schema = dj.schema('pipeline_experiment', locals(), create_tables=False)
+schema = dj.schema('pipeline_experiment', locals())#, create_tables=False)
```

Comment:
`locals()` is an old syntax, please replace any/every `dj.schema` with a version like this, without any `locals()`

```python
schema = dj.schema('pipeline_experiment')
```

Reply:
Also generally don't keep changed/removed code as comments. We have Git for being able to go back as necessary :)